tests/actions/createLabOrder.test.js

```diff
@@ -0,0 +1,80 @@
+import createLabOrder from '../../app/js/actions/createLabOrder';
+
+describe('createLabOrder action test-suite', () => {
+  beforeEach(() => moxios.install());
+  afterEach(() => moxios.uninstall());
+
+  it(`creates CREATE_LAB_ORDER_LOADING and CREATE_LAB_ORDER_FAILURE
+  action types upon error response from server`, async (done) => {
+    const errorData = {
+      error: {
+        message: 'invalid submission',
+      },
+    };
+
+    const request = moxios.requests.mostRecent();
+    moxios.stubRequest(`${apiBaseUrl}/encounter`, {
+      status: 401,
+      response: errorData,
+    });
+
+    const order = {
+      order: 'testorder',
+      concept: 'fkhfffkfjhf'
+    }
+
+    const expectedActions = [
+      {
+        type: 'CREATE_LAB_ORDER_LOADING',
+        status: true,
+      },
+      {
+        type: 'CREATE_LAB_ORDER_LOADING',
+        status: false,
+      },
+      {
+        type: 'CREATE_LAB_ORDER_FAILURE',
+        error: 'invalid submission',
+      },
+    ];
+
+    const store = mockStore({});
+    await store.dispatch(createLabOrder(order));
+    expect(store.getActions()).toEqual(expectedActions);
+    done();
+  });
+
+  it(`creates CREATE_LAB_ORDER_LOADING and CREATE_LAB_ORDER_SUCCESS
```

Comment:
use a proper description for this test cos someone else would want to refactor and have to trace through the actions and reducer to see what's actually happening

Reply:
If you look at the next line, you'll see that the description continues and I think it is quite descriptive as it is, please take a look at the next line, thank you.